internal/scorecard/alpha/scorecard.go

```diff
@@ -0,0 +1,117 @@
+// Copyright 2020 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package alpha
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"strings"
+
+	"github.com/operator-framework/operator-sdk/version"
+	"github.com/sirupsen/logrus"
+	"gopkg.in/yaml.v2"
```

Comment:
```suggestion
	"github.com/ghodss/yaml"
```

Reply:
The module `"github.com/ghodss/yaml"` has just 1 release and 14 contributors. See: github.com/ghodss/yaml
So, I do not think that we should use it. -1 for the suggestion. 
Also, see that we can use the `"gopkg.in/yaml.v3"` instead of. 

What is the motivation to do this change @jmccormick2001 @estroz ?

Reply:
Yeah, looks like `gihtub.com/ghodss/yaml` is unmaintained. But the Kubernetes community _is_ maintaining a fork at [`sigs.k8s.io/yaml`](https://github.com/kubernetes-sigs/yaml).

Here's the [rationale](http://web.archive.org/web/20190603050330/http://ghodss.com/2014/the-right-way-to-handle-yaml-in-golang/).

This is out-of-scope for this PR though. If everyone agrees to move all usage to `sigs.k8s.io/yaml`, we should do it in a separate PR.

Reply:
ok, I'll stick with gopkg.in/yaml.v2 for now, when the team changes the standard I'll update this.

Reply:
@joelanford the blog post which has the reason to use the third-party not maintained lib is from November 29, 2014. So, cannot it be an outdated info? 

I think we need to check if the GO pkgs are still facing some issue or not that would affect us. See that its latest versions have been used a lot: https://pkg.go.dev/search?q=yaml (V2 and V3)

I agree that it would be another PR, can we track it in an issue in the repo?

Reply:
done a pr: https://github.com/operator-framework/operator-sdk/pull/2786

Reply:
```suggestion
	"sigs.k8s.io/yaml"
```